runtime-common/search-index.ts

```diff
@@ -490,6 +526,45 @@ export class SearchIndex {
     let pathname = this.realmPaths.local(url);
     return ignore.test(pathname).ignored;
   }
+
+  private async getExternalCardDefinition(
+    moduleURL: URL,
+    ref: CardRef
+  ): Promise<CardDefinition | undefined> {
+    if (!baseRealm.inRealm(moduleURL)) {
+      // TODO we need some way to map a module to the realm URL that it comes from
+      // so that we now how to ask for it's cards' definitions
+      throw new Error(`not implemented`);
+    }
```

Comment:
@ef4 just a note something that we'll need to think about. I think this means that the search index needs to know about all the different realms cards that it's indexing might come from.

Reply:
try this URL pattern for _typeOf: `origin.com/path/to/module/_typeOf/export` which returns realm info (hangs off of meta)